front-end/src/App.jsx

```diff
@@ -9,12 +9,20 @@ import MainPage from './components/MainPage/MainPage.js';
 import { useAuth0 } from '@auth0/auth0-react';
 import { Link, Route, Switch } from 'react-router-dom';
 
+
+//delete
+import UserCookbook from './components/UserCookbook/UserCookbook.js';
+
 const App = () => {
 
   const { user, isAuthenticated, isLoading } = useAuth0();
 
   return (
     <>
+      {/* DELETE */}
+      <UserCookbook/>
```

Comment:
Did you wanna delete these before submitting? :P

Reply:
LOL, yes I did! Let me delete those XD